### DISCUSSION TITLE
What could be the reason that those two fields don't show the same value ?

### DISCUSSION THREAD

**Discussion by Robert-Riedl** [firefly-iii/firefly-iii#9362] [General]
![image](https://github.com/user-attachments/assets/2379cedd-9d38-4921-b3e8-91c03df00b20)

Yes, they are both the same (main asset) account - the left is the correct value. The right, I don't know how this comes to be.

**Comment by JC5** [comment]
Hey, thanks for opening an issue.

All I can think of is a cache issue. What happens when you browse to `/flush`?

**Comment by Robert-Riedl** [comment]
/flush doesn't bring a change - but I'm on the development branch (because of the API limit stuff) - maybe I should switch back to stable ?

**Comment by Robert-Riedl** [comment]
ah that field shows all transactions between first and last month !
